core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/version/VersioningConfiguration.java

```diff
@@ -35,20 +35,23 @@ public final class VersioningConfiguration {
 	private final String description;
 	private final LocalDate effectiveTime;
 	private final boolean force;
+	private final boolean versioningAsChildResource;
 	
 	public VersioningConfiguration(
 			String user,
 			ResourceURI resource,
 			String version, 
 			String description,
 			LocalDate effectiveTime,
-			boolean force) {
+			boolean force,
+			boolean versioningAsChildResource) {
```

Comment:
The verb `versioning` might be redundant here. This is the __Versioning__ Configuration, if the parameter would be e.g. `asChildResource` I think the operation itself is implied.

Reply:
Changing it to `childResource` as isAsChildResource accessor is a bit weird 😄.